### ISSUE TITLE
Says "ApplePersistence=NO" for all use

### ISSUE THREAD

**Issue by forthrin** [julienXX/terminal-notifier#267]
- [X] I looked in closed issues and it has not already been answered
- macOS: 10.12.6
- terminal-notifier: 2.0.0
- installation: homebrew

Since recently, I'm getting an `ApplePersistence=NO` messages for ALL use, even just for a plain `terminal-notifier -title foo -message bar` from the command line.

I've used the same terminal-notifier and macOS version for a long time.

This may be very well be caused by running `defaults write -g ApplePersistence -bool NO` recently. But why does `terminal-notifier` need to complain about this? How can I get rid of the complaint?

**Comment by DougThwaites** [comment]
Hi @forthrin 

I have had the same thing for same reason by disabling auto-save. To get things back to normal I ran:

`defaults delete -g ApplePersistence`

I assume this will undo any auto-save changes we have made but i'd rather that then having to fork and patch terminal notifier or find a replacement and even then I think it's more MacOS than terminal-notifier.

All credit goes to:

[Mentalist](https://apple.stackexchange.com/users/123184/mentalist) on the [Apple Stack Exchange](https://apple.stackexchange.com/questions/27544/how-to-completely-disable-auto-save-and-versions-in-mac-os-x-lion).

**Comment by forthrin** [comment]
Yep. It happens for other apps too. A (generic) workaround would be nice though, ie. a program that makes macOS not give this unnecessary complaint to the terminal. I don't know if this is so embedded to the core of macOS that it can't be patched even with a low-level C program?

**Comment by forthrin** [comment]
Seems to have gone away
